rundeckapp/grails-app/views/framework/projectNodeSources.gsp

```diff
@@ -82,7 +82,7 @@
                         </a>
                       </li>
 
-                    <feature:enabled name="enhanced-nodes">
+                    <feature:enabled name="enhancedNodes">
```

Comment:
maybe the feature:enabled tag lib can accept Feature enum type attribute e.g. `<feature:enabled feature="${Features.ENHANCED_NODES}">`

Reply:
I think we may need to reevaluate the Features enum. It's not a very flexible way to handle referencing features. If we are looking at adding and removing features at runtime I think we need something more flexible.

Reply:
hmm, perhaps a Feature bean type? the specific feature name can be a constant string in the implementation class